config/routes.rb

```diff
@@ -2,5 +2,11 @@
   get 'travel_time/get'
   resources :position_imports, only: [:create]
 
+  scope 'api' do
+    resources :rehire_sites do
```

Comment:
It's a message to others that use the program that you aren't using those routes. My typical workflow is something like get the pull request, run `rake routes` and see the list of available routes. Then test each endpoint. At that point if you use resources without only then it shows me all the endpoints as being available and then I don't realize there's nothing there to test until I peek into the controller.